src/Server.js

```diff
@@ -0,0 +1,83 @@
+/* @flow */
```

Comment:
I think there may not be much value in splitting out the HTTP server. I think most of the code is in custom-ish stuff around setting up the Express app. Going from there to the server is nearly trivial.

Reply:
I agree, the only real value here is that it's properly typed to expect and create the right context for the GQL middleware. The rest was meh in terms of reuse

Reply:
Do we still want/need this here? I think we can drop it now.

Reply:
ya i'm removing it